docs/src/examples/components/Provider/Usage/ProviderExampleTargetFrame.steps.ts

```diff
@@ -0,0 +1,7 @@
+const config: ScreenerTestsConfig = {
+  steps: [
+    (builder, keys) => builder.keys('body', keys.tab).snapshot('Focuses attachment in iframe'),
```

Comment:
cool!

Reply:
Yep, it should be cool, but it's not working 😭 

As we need this fix I will merge this PR and then we can investigate how to move focus for Screener. At least, now we can be sure that styles are rendered.